docx.go

```diff
@@ -52,32 +68,59 @@ func ConvertDocx(r io.Reader) (string, map[string]string, error) {
 					meta["CreatedDate"] = fmt.Sprintf("%d", t.Unix())
 				}
 			}
-
-		case f.Name == "word/document.xml":
-			textBody, err = parseDocxText(f)
+		case override.ContentType == "application/vnd.openxmlformats-officedocument.wordprocessingml.document.main+xml":
+			body, err := parseDocxText(f)
 			if err != nil {
 				return "", nil, err
 			}
-
-		case reHeaderFile.MatchString(f.Name):
-			header, err := parseDocxText(f)
+			textBody += body + "\n"
+		case override.ContentType == "application/vnd.openxmlformats-officedocument.wordprocessingml.footer+xml":
+			footer, err := parseDocxText(f)
 			if err != nil {
 				return "", nil, err
 			}
-			textHeader += header + "\n"
-
-		case reFooterFile.MatchString(f.Name):
-			footer, err := parseDocxText(f)
+			textFooter += footer + "\n"
+		case override.ContentType == "application/vnd.openxmlformats-officedocument.wordprocessingml.header+xml":
+			header, err := parseDocxText(f)
 			if err != nil {
 				return "", nil, err
 			}
-			textFooter += footer + "\n"
+			textHeader += header + "\n"
 		}
-	}
 
+	}
 	return textHeader + "\n" + textBody + "\n" + textFooter, meta, nil
 }
 
+func getContentTypes(f *zip.File) (*Type, error) {
```

Comment:
This naming seems a little ambiguous?  `getContentTypes` returns a `*Type`... Maybe Type should be called something else?